src/data.js

```diff
@@ -23,6 +23,9 @@ export const quizData = {
       },
       correct: 'b',
       selected: null,
+
+      isAnswerCorrect: true,
```

Comment:
I think it wouldn't be wise to use true as the default value. It might be better to store what answer the user chose. Like userAnswer: 'a', and then when you refresh the page, you know which answer user chose. Because otherwise, how does the application know to add correct or wrong class names to relevant options when the user refreshes the page?

Reply:
Approving your PR but please give a thought on this.